test/unit/logic/transaction.js

```diff
@@ -415,7 +415,7 @@ describe('transaction', () => {
 		it('should throw an error with no param', () => {
 			return expect(
 				transactionLogic.checkConfirmed.bind(transactionLogic)
-			).to.throw('Callback must be a function');
+			).to.throw('"callback" argument must be a function');
```

Comment:
Assuming you're running node v10, this shouldn't be changed.

Reply:
Good catch was running node 8.14. Done. Resolving.